src/parse.rs

```diff
@@ -167,13 +178,13 @@ mod tests {
 
     pub fn convert_to_conc_enum_maps(
         enum_maps: &HashMap<usize, HashMap<String, u32>>,
-    ) -> Arc<DashMap<usize, Arc<DashMap<String, u32>>>> {
+    ) -> Arc<DashMap<usize, Arc<DashMap<String, (u32, usize)>>>> {
         let c_enum_maps = Arc::new(DashMap::default());
 
         for (column, map) in enum_maps {
-            let c_map = Arc::new(DashMap::<String, u32>::default());
+            let c_map = Arc::new(DashMap::<String, (u32, usize)>::default());
             for (data, enum_val) in map {
-                c_map.insert(data.clone(), *enum_val);
+                c_map.insert(data.clone(), (*enum_val, 0_usize));
```

Comment:
Does that mean we are setting "unmap" as the default state for entries in the `enum_map`?

Reply:
It is called right after enum_maps in the saved model is loaded. In this case, though the enum's count is set to 0, the mapping is still effective and counted if the enum value will appear in coming data.

Reply:
Actually, this function exists in REmake, but I copied it here for just test codes.

Reply:
If in the new coming data, one of the previously "frequent" enum value appeared "infrequently", it will become "unmap", is that correct? If so, then is that what is intended?

Reply:
Only frequent enums are saved in the model but without frequency. Within only data being currently analyzed, frequency is calculated and frequent enums are mapped. It means saved map in the model is almost meaningless. I want your fresh idea regarding this.

Reply:
How about saving the counts, that is, changing the HashMap<usize, HashMap<String, u32>> to HashMap<usize, HashMap<String, (u32, usize)>? It involves some complexity, however, on the good side, it will make it consistent.

Reply:
Another option is to save the full list of mapped enum values with frequency in the model and use it in new coming data. Is this preferred?

Reply:
During writing the above, you wrote first. I will do as you want.

Reply:
Are you going to change this one or later in remake? Since as you said this is a testing code, I'm OK to merge it~

Reply:
I will change this after adding codes in remake.